### ISSUE TITLE
Copy Edits to Registration page [13,8,1.63]

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#2199]
Sign up page needs edits because of outdated text: https://technovation-qa.herokuapp.com/signup#/data-use

Updates start on Page 95 of https://docs.google.com/document/d/1NUqsJVrKDsgZ16t5wjVBrQDD8et6j9Y3VQWrIoEpHXc/edit#

- [ ] Data Use terms (95)
- [ ] Birthdate
- System should be calculating World Pitch age for August 1, 2020
- Date edits for both division explanations --> August 1, 2020

![Screen Shot 2019-09-16 at 12 35 47 AM](https://user-images.githubusercontent.com/22801026/65115077-4eb32680-d99c-11e9-8fdf-0a2adda047ad.png)

**Comment by hellafitz** [comment]
@rsgonzal what's blocked on this?

**Comment by rsgonzal** [comment]
I don't think it's blocked, but I did have the wrong screenshot in there.

**Comment by stenington** [comment]
I've made it so that Aug 1 cutoff date just has to be updated in one place each year, but it still does have to be updated in the codebase for now. 

I wanted to have it set as config and passed from the backend to the frontend, but I'm not well-versed enough in how the Vue apps initialize themselves to do it within the effort estimation I made here. I'll open a new ticket for that, if we ever want to do it, with some notes for myself.

**Comment by stenington** [comment]
#2222 written for the optional config follow up work

**Comment by rsgonzal** [comment]
Recommended QA:
- create a new account
- read through the Data Use terms looking for typos/make sure appropriate changes were made
- try out some different birthdates
- Make sure it is calculating age correctly for 2020 and says 2020
- make sure division definitions have the correct date

@nickiredescent do you have some time to QA this by Friday?

I did a first pass and looks good to me.

**Comment by nickiredescent** [comment]
The Data Use page and Birthdate calculations look good. Nothing seems out of place. Thanks!
